module/observations/helper.js

```diff
@@ -1972,4 +1996,98 @@ module.exports = class ObservationsHelper {
         });
     }
 
-};
\ No newline at end of file
+};
+
+/**
+  * Update Report Information In Solutions.
+  * @method
+  * @name _addReportInformationInSolution 
+  * @param {String} solutionId - solution id.
+  * @param {Object} userProfile - user read response
+  * @returns {Object} Solution information.
+*/
+
+function _addReportInformationInSolution(solutionId = "", userProfile = {}) {
+    return new Promise(async (resolve, reject) => {
+        try {
+
+            if ( userProfile && solutionId ) {
+
+                let district = {};
+                let organisation = [];
+
+                for (const location of userProfile["userLocations"]) {
```

Comment:
@priyanka-TL - we should make the code so that in future user can have multiple districts in profile and this logic works. check the same in projects too.